.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Linter
+        uses: pre-commit/action@v3.0.1
```

Comment:
the check must be only done on `docs` and `blog` dirs in order to avoid side effect on documentation READMEs

see https://github.com/zatsit-oss/zats-blog-content/actions/runs/8524560825/job/23349419957